common/libs/wallet/deterministic/trezor.ts

```diff
@@ -42,8 +41,24 @@ export class TrezorWallet extends HardwareWallet implements IFullWallet {
 
   public signRawTransaction(tx: EthTx): Promise<Buffer> {
     return new Promise((resolve, reject) => {
-      const { chainId, ...strTx } = getTransactionFields(tx);
+      const txFields = getTransactionFields(tx);
+
+      if (process.env.BUILD_ELECTRON) {
```

Comment:
It would be great to make a class that extends HardwareWallet that does this for you. @HenryNguyen5 how would you suggest this best be done?

Reply:
I'd probably just make a utility HOF instead that does the branching for you